cli/version.go

```diff
@@ -8,7 +8,7 @@ import (
 
 // versionDescription is used to describe version command in detail and auto generate command doc.
 // TODO: add description
-var versionDescription = ""
+var versionDescription = "Display the information of pouch version"
```

Comment:
We can add more details about client side and daemon side. And what kind of version will be displayed in output.

Reply:
But based on the example "
GoVersion:       go1.9.1
 KernelVersion:
 Os:              linux
 Version:         0.1.0-dev
 APIVersion:      1.24
 Arch:            amd64
 BuildTime:       2017-12-18T07:48:56.348129663Z
 GitCommit: ",
it seems that the information is just about one single side, not about client side and daemon side. 
In addition, you think that we should list detailed versions like GoVersion and KernelVersion?